### PR TITLE
Wording adjustments (and more)

### PR DESCRIPTION
**Pull request by rschwammborn**
Slightly bigger PR coming up:

1. Biggest changes include the standardization of words and sentences , e.g.: default options, german headings, removal of unnecessary $ signs and so on (these are the commits from April)
2. Furthermore, some updates in CSS, Simple Search and WMS loader were made, e.g. I removed a deprecated CSS example and adjusted screenshots (these are in the commits from March).

All these changes should already apply to Mapbender 3.2, thus I'm creating a open PR (correct me if I'm wrong).

### CODE REVIEW

File: de/functions/basic/feature_info.rst [hunk 1]
```diff
@@ -127,11 +129,11 @@ YAML-Definition:
...
    deactivateOnClose: true        # true/false, um die Funktion nach dem Schließen des Ergebnisfensters zu deaktivieren (Standard: true)
-   onlyValid: false               # Korrekte HTML Ausgabe erfordern. Standardwert ist false.
+   onlyValid: false               # Korrekte HTML Ausgabe erfordern (Standard: false).
    printResult: false             # Anzeige eines Links, über den die Infoabfrage ausgedruckt werden kann (Standard: false)
    displayType: tabs              # tabs/accordion (Standard: tabs)
-   width: 700                     # Breite des Dialogs in Pixel, Standardwert: 700
-   height: 500                    # Höhe des Dialog in Pixel, Standardwert: 500
+   width: 700                     # Breite des Dialogs in Pixel (Standard: 700)
+   height: 500                    # Höhe des Dialog in Pixel (Standard: 500)
    maxCount: 100	              # Maximale Anzahl an Treffern/Ergebnissen, die angezeigt werden soll
```
Comment (astroidex) [inline de/functions/basic/feature_info.rst:137]: new is hover color. Is not listed yet: featureColorHover: #00ff00

File: de/functions/basic/map.rst [hunk 2]
```diff
@@ -51,14 +51,16 @@ Weiterhin kann das Feld *Max. Extent* **(3)** definiert werden. Dieses gibt den
 YAML-Definition:
 ----------------
 
+Diese Vorlage kann genutzt werden, um die Karte in einer YAML-Anwendung einzubinden.
+
 .. code-block:: yaml
 
    layerset: null             # verweist auf ein Layerset, definiert zuerst das layerset und verweist darauf
-   dpi: 72                    # Auflösung, Standard ist 72
+   dpi: 72                    # Auflösung (Standard: 72)
```
Comment (astroidex) [inline de/functions/basic/map.rst:59]: dpi kann nicht mehr angegeben werden

File: de/functions/basic/map.rst [hunk 3]
```diff
@@ -51,14 +51,16 @@ Weiterhin kann das Feld *Max. Extent* **(3)** definiert werden. Dieses gibt den
...
    srs: "EPSG:4326"           # Koordinatenbezugssystem. Zwei Arten der SRS Definition werden unterstützt:
                                 - "EPSG:CODE" oder
                                 - "EPSG:CODE|MEIN SRS TITEL"
-   units: "degrees"           # Einheiten in Grad oder Meter, Standard ist "degrees" (Grad)
+   units: "degrees"           # Einheiten in Grad oder Meter (Standard: degrees)
```
Comment (astroidex) [inline de/functions/basic/map.rst:63]: units kann nicht mehr angegeben werden

File: de/functions/misc/wms_loader.rst [hunk 4]
```diff
@@ -19,30 +19,31 @@ Konfiguration
...
 
 Hinzufügen eines WMS über einen definierten Link
 ================================================
 
-Mapbender kann ein WMS über einen definierten Link hinzugefügt werden, z.B. zum Beispiel über :ref:`die Informationsabfrage<feature_info_de>` oder über Suchergebnisse.
-
-Aktivieren Sie in dem WMS Loader die Option **use Declarative** (setzen Sie in einer YAML-Anwendung die Option ``useDeclarative`` auf true.
+Mapbender kann einen WMS über einen definierten Link hinzufügen, z. B. über :ref:`die Informationsabfrage<feature_info_de>` oder über Suchergebnisse.
 
+Dazu muss in "WMS laden" die Option **use Declarative** aktiviert werden. In einer YAML-Anwendung wird die Option ``useDeclarative`` auf true gesetzt.
```
Comment (astroidex) [inline de/functions/misc/wms_loader.rst:46]: Satz kann entfallen

File: en/functions/basic/feature_info.rst [hunk 5]
```diff
@@ -116,11 +116,13 @@ The necessary adjustment is shown here using a configuration for MapServer. In t
...
+
 .. code-block:: yaml
 
    title: FeaureInfo              # title of the element
    tooltip: Feature Info          # text to use as tooltip
-   type: dialog                   # Default: dialog.
+   type: dialog                   # defines the element type (default: dialog)
    target: map                    # Id of Map element to query
    autoActivate: false            # true/false open when application is started (default: false)
    deactivateOnClose: true        # true/false to deactivate the functionality after closing the result dialog (default: true)
```
Comment (astroidex) [inline en/functions/basic/feature_info.rst:128]: missing parameter                        highlighting: true
                        featureColorDefault: #f0f0f0
                        featureColorHover: #00ff00

**Review by astroidex — COMMENTED** [review]
Loos good. Only some comments

**Comment by rschwammborn** [comment]
Made some more commits:

- Fixes to all of @astroidex comments
- Removal of 'Redlining' wording artefacts
- Replace of 'man' with new sentences

**Review by astroidex — APPROVED** [review]

**Pull request merged**